src/main/java/telraam/logic/positioner/nostradamus/v2/Nostradamus.java

```diff
@@ -0,0 +1,149 @@
+package telraam.logic.positioner.nostradamus.v2;
+
+import org.jdbi.v3.core.Jdbi;
+import telraam.database.daos.BatonSwitchoverDAO;
+import telraam.database.daos.PositionSourceDAO;
+import telraam.database.daos.StationDAO;
+import telraam.database.models.*;
+import telraam.logic.positioner.Position;
+import telraam.logic.positioner.PositionSender;
+import telraam.logic.positioner.Positioner;
+
+import java.util.*;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentLinkedQueue;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+
+public class Nostradamus implements Positioner {
+    private static final Logger logger = Logger.getLogger(Nostradamus.class.getName());
+    private final String SOURCE_NAME = "nostradamus";
+    private final int MIN_RSSI = -84; // Minimum rssi strength that a detections needs to have
+    private final int INTERVAL_FETCH_MS = 10000;
+    private final int INTERVAL_UPDATE_MS = 200;
+    private final int LENGTH_OFFSET = 40; // Distance from the last station to the finish in meter
```

Comment:
This is currently hardcoded, but is dependent on the physical setup; is there somewhere else we can get this number from? Maybe a total lap length, and subtract the last station position?

Reply:
With the current data, no. Unless we always place the last station on the finish line.
However I've moved it to a config variable.